### PR TITLE
Link the "Contributing Guide" to the README and place in the documenation

### PR DESCRIPTION
**Pull request by travishathaway**
I recently noticed that the "Contributing Guide" is a little difficult to find (I totally missed seeing it in the root of the repository!). In order to make easier to find, this pull request adds this as a page to the official documentation and also links to it from the README file.

### CODE REVIEW

File: docs/source/contributing-guide.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+```{include} ../../CONTRIBUTING.md
+```
+[contributing-guide]: contributing-guide
```
Comment (beeankha) [inline docs/source/contributing-guide.md:3]: Oh nice, so we can see the CONTRIBUTING.md file directly on `conda-build` docs this way! What a great solution to keep everything in one place yet accessible from multiple places!

File: README.rst [hunk 2]
```diff
@@ -68,3 +68,10 @@ The documentation for ``conda`` is at http://conda.pydata.org/docs/. You can
 subscribe to the `conda mailing list
 <https://groups.google.com/a/continuum.io/forum/#!forum/conda>`_.  The source
 code and issue tracker for ``conda`` are on `GitHub <https://github.com/pydata/conda>`_.
+
+Contributing to conda-build
+---------------------------
+
+We always welcome new contributors for ``conda-build``, especially for bug-fixes and
```
Comment (beeankha) [inline README.rst:75]:
```suggestion
We always welcome new contributors for ``conda-build``, especially for bug fixes and
```

Just a nitpick; I don't think we need the hyphen there

**Review by beeankha — APPROVED** [review]
🚀 Thanks for doing this, Travis!

File: README.rst [hunk 3]
```diff
@@ -68,3 +68,10 @@ The documentation for ``conda`` is at http://conda.pydata.org/docs/. You can
...
+new feature requests. To help get your environment setup and begin developing, please
```
Comment (kathatherine) [inline README.rst:76]: "set up" is a verb, "setup" is a noun, for future reference. Same with sign in, log in, etc.

```suggestion
new feature requests. To help get your environment set up and begin developing, please
```

File: docs/source/contributing-guide.md (hunk 1 above)
Comment (kathatherine) [reply docs/source/contributing-guide.md:3]: Love it.

**Review by kathatherine — CHANGES_REQUESTED** [review]
One change. Thanks!

File: docs/source/contributing-guide.md (hunk 1 above)
Comment (travishathaway) [reply docs/source/contributing-guide.md:3]: @beeankha , this is how it's handled in the `conda` repository 😄

**Pull request merged**